### ISSUE TITLE
 Brave icon is not shown on the taskbar

### ISSUE THREAD

**Issue by MadhaviSeelam** [brave/brave-browser#27558]
## Steps to Reproduce

   1.  Install 1.47.151 
   2.  launch Brave

## Actual result:

Taskbar shows white page like icon

## Expected result:
Brave icon should be displayed

## Reproduces how often: 

More frequently 

## Brave version (brave://version info)

```
Brave | 1.47.151 Chromium: 109.0.5414.46 (Official Build) beta (64-bit)
-- | --
Revision | 6e36b77363ef3febbe792af680fa1367993ddcf0-refs/branch-heads/5414@{#709}
OS | Windows 11 Version 22H2 (Build 22621.963)
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:
 @rebron @nullhook 
cc: @kjozwiak

**Comment by Nihal-Raj** [comment]
@MadhaviSeelam It can be solved by **right clicking on the blank logo** -> **right clicking on Profile _x_ - Brave** -> **Properties** -> **Change Icon**, selecting the desired icon, and clicking `Apply`, this will solve the problem.

Hope it works

**Comment by Crabmeat1432** [comment]
Changing the icon didn't help, but when I pinned the icon to my desktop, the icon came back.  Then I unpinned it and it stayed, for the time being at least.

**Comment by kjozwiak** [comment]
> Changing the icon didn't help, but when I pinned the icon to my desktop, the icon came back. Then I unpinned it and it stayed, for the time being at least.

Believe this only happens when you're doing a clean install and we auto-pin to the `taskbar`. I tried installing Chrome and it didn't attempt to auto-pin so the issue didn't occur. I'm not 100% sure when Google auto-pins. But I haven't see the issue happen in a while. @rebron maybe we need to get someone to take a look as this is pretty annoying for `Win` users as it happens on new installs. Probably not good for retention/noticing broken icons on the `taskbar`.

**Comment by bsclifton** [comment]
+1 from @thypon - basically if you do an install and it does the `Pin Brave to Taskbar?` and you click `Yes` twice, this will happen.

**Comment by bsclifton** [comment]
I don't think this is an upstream issue @rebron @kjozwiak - I couldn't reproduce in Chrome as it doesn't offer a way to pin to taskbar. There is an issue captured upstream however for if you pin a PWA where it can lose the icon:
https://bugs.chromium.org/p/chromium/issues/detail?id=1006568
